modules/ROOT/pages/6.8.1-release-notes.adoc

```diff
@@ -0,0 +1,556 @@
+= TinyMCE 6.8.1
+:navtitle: TinyMCE 6.8.1
```

Comment:
Since we never released 6.8.0 shouldn't this include all the items for 6.8.0 as well? Such as the new `default_font_stack` option?

Reply:
@spocke the release notes have all tickets in `scope` for `6.8.0` and `6.8.1`, but the confusion was that i thought you had merged a PR for the new `default_font_stack` option, but just remembered that you created a `draft` in the Jira ticket.  I will put this together soon- thanks for catching it early.

Reply:
Added in [this pr](https://github.com/tinymce/tinymce-docs/pull/3021), for new `default_font_stack` option in the `6.8.1` release notes.